packages/jest-haste-map/src/lib/docblock.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+'use strict';
+
+const commentEndRe = /\*\/$/;
+const commentStartRe = /^\/\*\*/;
+const docblockRe = /^\s*(\/\*\*(.|\r?\n)*?\*\/)/;
+const ltrimRe = /^\s*/;
+const multilineRe = /(?:^|\r?\n) *(@[^\r\n]*?) *\r?\n *([^@\r\n\s][^@\r\n]+?) *\r?\n/g;
+const propertyRe = /(?:^|\r?\n) *@(\S+) *([^\r\n]*)/g;
+const stringStartRe = /(\r?\n|^) *\*/g;
+const wsRe = /[\t ]+/g;
+
+function extract(contents) {
+  const match = contents.match(docblockRe);
+  return match ? match[0].replace(ltrimRe, '') || '' : '';
```

Comment:
use 
```js
const docblockRe = /^\s*(\/\*\*([^]*?)\*\/)/;
```
and `match[1]`. That will give you the docblock without leading `/**` and trailing `*/`. Also, `|| ''` should be unecessary.